clang/test/CodeGenCXX/pr137276.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clang_cc1 -std=c++20 -triple x86_64-linux-gnu -emit-llvm -o - %s | FileCheck %s
+
+using ulong = unsigned long;
+template <class... Ts>
+void g(Ts... args) {
+  ulong arr[3] = {ulong(args)...};
+  (void)arr;
+}
+extern void f() {
+  g(nullptr, 17);
+}
+
+// CHECK: {{^}}  store i64 0, ptr %arr, align 8{{$}}
```

Comment:
- Please just add this to the existing `test/CodeGenCXX/nullptr.cpp` test.
- Please use `CHECK-LABEL` to make sure that your CHECK line is within the function you intend for it to be in.
- You don't really need a template here; a simpler reduction is just:

```
void test(nullptr_t np, int i) {
  long arr[] = { long(np), i, 0 };
}
```